lib-extra/src/main/java/com/diffplug/spotless/extra/java/EclipseFormatterStep.java

```diff
@@ -47,14 +40,14 @@ private EclipseFormatterStep() {}
 	private static final String FORMATTER_METHOD = "format";
 
 	/** Creates a formatter step for the given version and settings file. */
-	public static FormatterStep create(File settingsFile, Provisioner provisioner) {
-		return create(defaultVersion(), settingsFile, provisioner);
+	public static FormatterStep create(Iterable<File> settingsFiles, Provisioner provisioner) {
```

Comment:
I'm not opposed to supporting multiple config files.

But what is the reason to deprecate the specialized method for the common use-case of a single config file?